### DISCUSSION TITLE
🙏 求助！！！透视表无法计算出总计

### DISCUSSION THREAD

**Discussion by chensuifeng** [antvis/S2#2638] [Q&A]
<script src="https://unpkg.com/@antv/s2@1.54.7/dist/index.min.js"></script>
配置：
![image](https://github.com/antvis/S2/assets/24535400/cddab63d-251a-4113-8d97-3384e6506a69)
结果：
![image](https://github.com/antvis/S2/assets/24535400/c102a5be-895d-47bc-8edc-75714306a354)

**Comment by wjgogogo** [comment]
你所使用的 1.x 的版本，calc 配置不对：https://s2-v1.antv.antgroup.com/api/general/s2-options#totals
![image](https://github.com/antvis/S2/assets/17964556/85283ae6-3e6d-43eb-828f-9feaa13d6308)
